index.rst

```diff
@@ -5,59 +5,108 @@ Technote Style and Writing Guide
+Technote abstracts should include a short 2-3 sentence explanation of the goal of the technote. Ideally, it should also include a 1-2 sentence summary of the main conclusions.
 
-Introduction
-================
+*Example: We verify that the pointing accuracy of the system is compliant with the requirement of 5 arcsec RMS. We do this by analyzing star tracker data during a soak test mimicking real scheduled observations. Our conclusions are that the pointing performance is compliant with the requirement, except at low (<30 deg.) elevation measurements.* 
 
-See the `Documenteer documentation <https://documenteer.lsst.io/technotes/index.html>`_ for tips on how to write and configure your new technote.
+*Example: We explore the problems found on 2024-02-01 on the azimuth drives to diagnose the issue. We study what TMA settings lead to the drives going to fault. Currently not correlation of velocity, acceleration, jerk or initial/final positions are found*.
 
-Main part
-=========
 
-Technote Style Guide
---------------------
-Text Parts
-^^^^^^^^^^
-Reasoning:
-Using the same structure in all technotes makes it easy to jump in to the needed information
+Introduction
+============
 
-A document should be structured in the following parts
+This is a short reference to provide guidance and support for creating technotes related to SITCOM activities. For details on how to start, configure and edit a technote, see the `Documenteer documentation <https://documenteer.lsst.io/technotes/index.html>`_. This includes adding yourself as author in the author database file, in case you are not there.
 
-- Introduction
-- Main Part
-- Summary and Conclusion
+The introduction should expand the details and context in which the technote is written, and why its goals are pursued. In the case of a very straightforward measurement, or self-exaplanatory, it could be merged with the abstract. Ideally these would be exceptions.
 
-Technote Writing Guide
-----------------------
-Writing tempus
-^^^^^^^^^^^^^^
-**Suggestion:**
+*Example: Vibrations of the TMA structure immediately after a slew can impact the image quality performance, if they prolong into the time in which data is collected. We need to measure the residual vibration of the TMA in various axes, and in identical conditions as in real observations. These tests should be combined with those that measure the jittering of other mechanical and optical systems in the light path for an overall assessment of the 
+optical performance. The results in this technote should be regarded as preliminary because the final TMA settings have not been configured.*
 
-Write in present tense as much as possible.
+Related Tickets
+===============
 
-**Reason:**
+A bulleted list of related tickets and their titles, including those issued to  writing the technote itself, those related to relevant code used to produce the results and any other Operations tickets that can help trace the problem. 
```

Comment:
I was also going to ask to specify what we mean by Operations tickets. +1 for the link suggestion if we meant OBS tickets, otherwise you could just open it up to say "any other tickets that can help trace the problem". 

Might also be worth adding an OBS ticket or DM ticket to the bulleted list of examples the follows.